news/_posts/2015-11-23-prestashop-core-team.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title:  "PrestaShop Core Team Organization"
+subtitle: "Who's Who"
+date:   2015-11-23 11:12:00
+author:  [ jnadaud ]
+icon: icon-organisation
+tags: [news, organisation, core, team]
+---
+
+### PrestaShop Core Organization
+
+PrestaShop Core members are divided into three groups. The privileges granted to a group are automatically granted to all higher priority groups.
+
+Those PrestaShop Core groups, in descending order of priority, are:
+
+* **Project Leaders**  
+Elects members in any other group;  
+Merges pull requests in all PrestaShop repositories.
+
+* **Mergers**
+Merge pull requests for some part of PrestaShop on which they have been granted privileges.  
+
+* **Deciders / Product manager**  
+Decide to merge or reject a pull request.
+
+### Active PrestaShop Core Members
+* **Project Leaders**:  
+Sebastien Levaillant ([slevaillant](https://github.com/orgs/PrestaShop/people/slevaillant))  
+
+* **Mergers**:  
+François Marie de Jouvencel ([djfm](https://github.com/orgs/PrestaShop/people/djfm)) can marge into Starter Theme, Front components, Tax components, I18n components   
+Julien Bourdeau ([julienbourdeau](https://github.com/orgs/PrestaShop/people/julienbourdeau)) can merge into Starter Theme and Core components  
+François Gaillard ([gaillafr](https://github.com/orgs/PrestaShop/people/gaillafr)) can merge into Core components  
+Luc Vandesype ([kelu95](https://github.com/orgs/PrestaShop/people/kelu95)) can merge into Back Office components and Architecture components  
+Xavier Gouley ([xGouley](https://github.com/orgs/PrestaShop/people/xGouley)) can merge into Back Office components and Architecture components  
+Julien Martin ([shudrum](https://github.com/orgs/PrestaShop/people/shudrum)) can merge into Back Office components and Architecture components  
+Maxime Biloé ([maximebiloe](https://github.com/orgs/PrestaShop/people/maximebiloe)) can merge into Starter Theme and Core components  
+Thibaud Chauvière ([tchauviere](https://github.com/orgs/PrestaShop/people/tchauviere)) can merge into Module components and Core components  
+Thomas Nabord ([Quetzacoalt91](https://github.com/orgs/PrestaShop/people/Quetzacoalt91)) can merge into can merge into Module components and Core components  
+Xavier Borderie ([xBorderie](https://github.com/orgs/PrestaShop/people/xBorderie)) can merge into PrestaShop developer blog and PrestaShop documentation  
+Alexandra Even ([AlexEven](https://github.com/orgs/PrestaShop/people/AlexEven)) can merge into PrestaShop developer blog and PrestaShop documentation     
+
+* **Deciders**:  
+Julien Bourdeau ([julienbourdeau](https://github.com/orgs/PrestaShop/people/julienbourdeau))  
+François Marie de Jouvencel ([djfm](https://github.com/orgs/PrestaShop/people/djfm))  
+Vincent Beudez ([vincentbz](https://github.com/orgs/PrestaShop/people/vincentbz))  
+Guillaume Bruère ([superGB](https://github.com/orgs/PrestaShop/people/vincentbz))  
```

Comment:
Thank you for proofreading, @PrestaEdit !
This is actually more of a thinking-in-progress, and we're not ready to finalize it.
I'm closing this PR for now.